app/assets/v2/js/grants/cancel_subscription.js

```diff
@@ -37,6 +37,8 @@ $(document).ready(() => {
 
         web3.eth.getAccounts(function(err, accounts) {
 
+          let url;
```

Comment:
Is this needed ?

Reply:
Yes, it is needed because we need to be able to pass the url from view.py to the js, but we dont want to immediately redirect if all transaction or ajax requests havent been completed. So this is a way to hold that url data. It doesnt need to be in as high of scope as it is here though. I will bring it down to a more appropriate level.